mono/mini/mini-runtime.h

```diff
@@ -510,11 +510,23 @@ void mono_enable_jit_map (void);
 void mono_emit_jit_map   (MonoJitInfo *jinfo);
 void mono_emit_jit_tramp (void *start, int size, const char *desc);
 gboolean mono_jit_map_is_enabled (void);
+#if defined(ENABLE_JIT_DUMP)
+#include <sys/mman.h>
+#include <sys/syscall.h>
```

Comment:
Is `syscall.h` needed by everything that includes `mini-runtime.h`, or only by `mini-runtime.c`?

Reply:
only runtime.c

Reply:
Please move the `#include` to `mini-runtime.c`, then.  (In general: you want to include only where you need the definition of something.  In this case, requiring `syscall.h` is an implementation detail, so whoever includes this header doesn't have to also get all the definitions found in `syscall.h`.)

Reply:
sure, make sense

Reply:
Do you need to include syscall.h? Try without it?

Reply:
Oh I see.